app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,32 @@
+.dash-unit {
+  margin-bottom: 30px;
+  padding:10px;
+  border: 1px solid #383737;
+  background-image: url(../img/sep-half.png);
```

Comment:
This file is not assigned to project.
It should be placed in app/assets/images...